community/salt/APKBUILD

```diff
@@ -1,67 +1,166 @@
 # Contributor: Francesco Colista <fcolista@alpinelinux.org>
+# Contributor: Olivier Mauras <olivier@mauras.ch>
 # Maintainer: Francesco Colista <fcolista@alpinelinux.org>
 pkgname=salt
-pkgver=2016.3.2
-pkgrel=0
-pkgdesc="A remote execution and communication system built on zeromq"
+pkgver=2016.3.3
+pkgrel=1
+pkgdesc="A parallel remote execution system"
 url="https://github.com/saltstack/salt"
 arch="noarch"
 license="ASL 2.0"
-depends="py-tornado py2-markupsafe py2-futures" 
+depends="py-tornado py2-yaml py-jinja2 py2-markupsafe py2-msgpack py2-crypto py2-zmq
+    py2-six py2-requests procps"
 depends_dev=""
-makedepends="$depends_dev python2-dev py-zmq
-	py2-crypto py2-psutil py-jinja2 py-yaml py-msgpack"
+makedepends="$depends_dev python2-dev py-tornado py2-yaml py-jinja2 
+    py2-markupsafe py2-msgpack py2-crypto py2-zmq py2-requests py2-futures"
 install=""
-subpackages="$pkgname-doc"
-source="https://github.com/saltstack/salt/releases/download/v$pkgver/$pkgname-$pkgver.tar.gz
-		salt-master.initd
-		salt-master.confd
-		salt-minion.initd
-		salt-minion.confd
-		salt-syndic.initd
-		salt-syndic.confd"
+subpackages="$pkgname-doc $pkgname-master $pkgname-minion $pkgname-syndic
+    $pkgname-api $pkgname-cloud $pkgname-ssh"
+source="$pkgname-$pkgver.tar.gz::https://codeload.github.com/saltstack/$pkgname/tar.gz/v$pkgver
+        salt-api.confd
+        salt-api.initd
+        salt-master.confd
+        salt-master.initd
+        salt-minion.confd
+        salt-minion.initd
+        salt-syndic.confd
+        salt-syndic.initd
+        0001-alpine-support.patch"
 
 builddir="$srcdir"/$pkgname-$pkgver
+prepare() {
+    local i
+    cd "$builddir"
+    for i in $source; do
+        case $i in
+        *.patch) msg $i; patch -p1 -i "$srcdir"/$i || return 1;;
+        esac
+    done
+}
```

Comment:
Please remove `prepare` function; applying of patches is already handled by the `default_prepare` function that is called when no custom `prepare` function is declared.